src/commands/openCmd.ts

```diff
@@ -1,25 +1,20 @@
 import { script_v1 } from 'googleapis';
-import {
-  loadAPICredentials,
-  script,
-} from './../auth';
+// setup inquirer
```

Comment:
It looks like we set up inquirer a lot.
Can you move it to a separate file and import it?

Reply:
Sure... but in another PR if you don't mind